restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -361,9 +362,9 @@
 	"projectLogResetLabel": "Reiniciar",

```

Comment:
Also some errors found in (355), change for:
"playStepLoopTooltip": "Cantidad de veces que el archivo de audio será reproducido. Por defecto es '1'. Con '0' el archivo se reproducirá repetidamente hasta que se cuelgue la llamada.",